libs/dev-dock/frontend/src/dev_dock.tsx

```diff
@@ -336,17 +364,29 @@ function UsbDriveMockControls() {
     clearUsbDriveMutation.mutate();
   }
 
+  const isFeatureEnabled = isFeatureFlagEnabled(
+    BooleanEnvironmentVariableName.USE_MOCK_USB
+  );
+
+  const disabled = !isFeatureEnabled || !getUsbDriveStatusQuery.isSuccess;
+
   const isInserted = status === 'inserted';
   return (
     <Column>
       <UsbDriveControl
         onClick={onUsbDriveClick}
         isInserted={isInserted}
+        disabled={disabled}
         aria-label="USB Drive"
       >
-        {status && <UsbDriveIcon isInserted={isInserted} />}
+        <UsbDriveIcon isInserted={isInserted} disabled={disabled} />
+        {!isFeatureEnabled && (
+          <UsbMocksDisabledMessage>
+            <p>USB mock disabled</p>
```

Comment:
If we wanted to add the info about the env var, I think an info icon with a tooltip would be a good solution. And we could use the same pattern for the smart card mocks disabled message.

Alternatively we could have an "Enable" button that pops up an alert with instructions (and eventually make an actual API for enabling/disabling mocks via the dev dock).

Reply:
OK, going to punt on tweaking what I have now but like your ideas.

Is it possible to enable/disable mocks while the app is running? Or would the dev dock kill the app and restart (rebuild)?

Reply:
That's a good question and I don't know the answer